packages/foundations/media-queries.ts

```diff
@@ -148,4 +148,5 @@ export {
 	desktop,
 	leftCol,
 	wide,
+	breakpointMap,
```

Comment:
This should be fine, but let's give it a friendlier name!

```suggestion
	breakpointMap as breakpoints,
```